### PR TITLE
stops poolcontroller drowning sending messages to admins

### PR DESCRIPTION
**Pull request by Kyep**
Currently, when anyone takes damage due to being underwater without proper breathing gear, online admins get a notification like this:

ATTACK: *invalid:/obj/machinery/poolcontroller*(?) (FLW) drowned CKEY/(CHARNAME)(?) (FLW) 

Such events can trigger every few seconds, which is very spammy.

This PR changes it so that:
- Players who are drowning no longer generate attack notifications to admins.
- The exception is SSD players. Those still generate attack notifications.

No changelog, as this change is invisible to players.

### CODE REVIEW

**Comment by GunDOSMk1** [comment]
@Kyep You are my number one person of the year for doing this. My bleeding eyes thank you.

**Comment by Anasari** [comment]
finally.

A better title would be "Stops poolcontroller from drowning the admin logs in messages"

**Comment by Spartan6** [comment]
SSD players should still appear and/or give a single warning. Civs sometimes drag SSD's into the pool for a laugh.

**Comment by Aurorablade** [comment]
Yeah I would like to know when people are drowning sometimes

**Comment by Kyep** [comment]
@Spartan6 @Aurorablade Edited. Now, it only notifies you if the person drowning is SSD.

File: code/game/machinery/poolcontroller.dm [hunk 1]
```diff
@@ -100,7 +100,10 @@
 		if(drownee.losebreath > 20)	//You've probably got bigger problems than drowning at this point, so we won't add to it until you get that under control.
 			return
 
-		add_logs(src, drownee, "drowned", null, null, 0)	//log it to their VV, but don't spam the admins' chats with the logs
+		if (isLivingSSD(drownee))
```
Comment (Fox-McCloud) [inline code/game/machinery/poolcontroller.dm]: spaces.

Comment (Kyep) [reply code/game/machinery/poolcontroller.dm]: Fixed

**Comment by Fox-McCloud** [comment]
@DarkPyrolord @Necaladun

**Comment by DarkPyrolord** [comment]
👍

**Pull request merged**